build_process/package-management.md

```diff
@@ -59,19 +59,19 @@ To quickly create a new package run `npm init -y --scope sparkbox`. This will cr
 }
 ```
 
-Private packages will typically be used in client work (design systems, component libraries, various utilities). Consider scoping these to something company-appropriate to avoid naming collisions with open-source packages.
+We typically use private packages in client work (design systems, component libraries, various utilities). Consider scoping these to something company-appropriate to avoid naming collisions with open-source packages.
 
-Additional fields that we will most likely want to use in package.json:
-- [module][pkgModule] - offer an ES module entry point
+These are additional fields that we usually want to use in package.json:
+- [module][pkgModule] — offer an ES module entry point
```

Comment:
I'm a little confused about the noun/verb agreement here. If these are talking about types of fields, I would guess the verb needs to be plural (the module type of field offers an ES...) as opposed to singular (the module type of field offer an ES...). Is that correct, or am I not quite understanding the context?